src/firmware/main/math/polynomial_1d.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+// Need this for size_t
+#include <stddef.h>
```

Comment:
I don't see `size_t` used anywhere in the header, only in the `.c` file. Can this include be moved there?

Reply:
Already removed this entirely.